### ISSUE TITLE
Many buffering with IA 2.3.8, downgrade to IA 2.3.4 fixes the problem

### ISSUE THREAD

**Issue by joaopa00** [peak3d/inputstream.adaptive#1260]
### Describe the bug
Here is a clear and concise description of what the problem is:

Many buffering with IA 2.3.8, downgrade to IA 2.3.4 fixes the problem

## Expected Behavior

No buffering

## Actual Behavior

With IA 2.3.8, many streams played with Catchuptvandmore buffer a lot. I and others Catchuptvandmore users have this problem

### To Reproduce
1.  With catchuptvandmore, try to play viatelepaese for instance

### Debuglog

The debuglog can be found here:

### MPD/M3U8s/ISM

[m3u8 file](https://srv.webtvmanager.fr:3118/live/viatelepaeselive.m3u8)

### Your Environment
Used Operating system:
 - [x] Android
 - [ ] iOS
 - [ ] tvOS
 - [x] Linux
 - [ ] OSX
 - [ ] Windows
 - [ ] Windows UWP

 - Operating system version/name:
 - Kodi version: Nexus 20.1
 - inputstream.adaptive version: 2.3.8

*note: Once the issue is made we require you to update it with new information should that be required.
Team Kodi will consider your problem report however, we will not make any promises the problem will be solved.*

**Comment by glennguy** [comment]
Quick question - is this a live stream?

**Comment by joaopa00** [comment]
Yes, it is a live stream

**Comment by glennguy** [comment]
Could you please enable debug logging and attach a log that captures the problem?

**Comment by glennguy** [comment]
I can access the stream from here (Australia), and the speed is incredibly slow i.e 
```
2023-05-17 19:33:10.821 T:23452   debug <general>: CurlFile::XFILE::CCurlFile::Open - <https://srv.webtvmanager.fr:3118/live/viatelepaeselive-75958.ts>
2023-05-17 19:33:39.910 T:23452   debug <general>: AddOnLog: inputstream.adaptive: Download finished: https://srv.webtvmanager.fr:3118/live/viatelepaeselive-75958.ts (downloaded 3401484 byte, speed 116933.00 byte/s)
```
29 seconds for 3.5MB.

Are you sure there isn't a problem with throttling from the provider?

**Comment by matthuisman** [comment]
@glennguy maybe post your new build to test?
Or does that only affect dash?

**Comment by joaopa00** [comment]
Indeed, the stream I gave has some problems. But buffering also happens for many others streams. Unfortunately, it is harder to test since link to fetch the mpd are tokenized.

**Comment by glennguy** [comment]
@joaopa00 can you please try these builds?:

Mac/Windows/Android: https://jenkins.kodi.tv/blue/organizations/jenkins/xbmc%2Finputstream.adaptive/detail/PR-1262/1/artifacts

Linux Arm (CoreELEC/LibreELEC) : [inputstream.adaptive.arm.20.3.8-mod.zip](https://github.com/xbmc/inputstream.adaptive/files/11520831/inputstream.adaptive.arm.20.3.8-mod.zip)

HLS is not changed since the 20.3.8 release but as Matt suggested DASH is what is affected.

**Comment by joaopa00** [comment]
I tried your build with TF1 (and related channels) live streams. The situation is a lot better. Tested on Windows and Android.

**Comment by glennguy** [comment]
Thanks @joaopa00 !

**Comment by CastagnaIT** [comment]
closed as fixed by #1268
